rakelib/utils.rake

```diff
@@ -68,6 +68,7 @@ class Utils
 
   # select the xcode version we want/support
   def self.version_select
+    Rake.sh 'mdimport /Applications/Xcode* && sleep 1'
```

Comment:
I really don't like that we have to hardcode an arbitrary delay there. Slows down the build and duration might vary anyway depending on machine and how many Xcode bundles the command are gonna indeed… I'm sure we can do better?